### ISSUE TITLE
Visual / Code editor switch buttons: indicate the state to assistive technologies

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5699]
Reported by one of the accessibility testers:

Switching from Visual editor to Code editor:
- When one of the editors is selected, the state “selected” is not announced. There is a visual feedback (a check icon) but none for screen readers. The only clue is that when an editor is not selected the ⌘+shift+Alt+M will be announced with the button label.

Also, it would be nice to add a message to confirm that the switch was made.

<img width="257" alt="screen shot 2018-03-20 at 00 36 16" src="https://user-images.githubusercontent.com/1682452/37628038-b2bc9c48-2bd7-11e8-9eb4-ac438ca7225a.png">

I guess for buttons the most appropriate ARIA attribute is aria-pressed https://www.w3.org/TR/wai-aria-1.1/#aria-pressed as aria-selected is not used for buttons https://www.w3.org/TR/wai-aria-1.1/#aria-selected
